docs/advanced/naming.md

```diff
@@ -1,190 +1,245 @@
+    !!!
+
+![#eed49f](https://placehold.co/14x14/eed49f/eed49f.png) Season/episode number
+:   The season and episode number
+
+    !!!info
+    For specials, use `S00EXX` for guaranteed automatic parsing and correct identification. *However, it is suggested to add an additional identifier such as including `OVA` or the episode title (i.e. `S00E01 - OVA Title`) to easily differentiate between episodes*
+    !!!
+
+![#a6da95](https://placehold.co/14x14/a6da95/a6da95.png) Version number [!badge variant="warning" text="Conditional"]
+:   The version number for patched releases in `vX` format, where `X` is the version number
+
+![#81c8be](https://placehold.co/14x14/81c8be/81c8be.png) Episode title [!badge variant="success" text="Optional"]
+:   The title of the episode. *We recommend using names found in databases such as [TheTVDB](https://www.thetvdb.com) (TV series) and [The Movie Database (TMDB)](https://www.themoviedb.org/?language=en-US) (movies)*
+
+    !!!warning
+    This should generally be skipped to avoid creating file paths larger than 256 characters, which may cause problems with various programs.
+    !!!
+
+![#91d7e3](https://placehold.co/14x14/91d7e3/91d7e3.png) Source type
+:   The source where the video/audio are taken from
+
+    !!!info
+    For Blu-ray/DVD remuxes, add `Remux` to the end (e.g. `BD Remux` or `DVD Remux`)
+    !!!
+
+![#8aadf4](https://placehold.co/14x14/8aadf4/8aadf4.png) Video resolution
+:   The vertical resolution of the file's video. *We recommend using [MediaInfo](https://mediaarea.net/MediaInfo) to find this information*
+
+![#c6a0f6](https://placehold.co/14x14/c6a0f6/c6a0f6.png) Video format
+:   The codec of the file's video. *We recommend using [MediaInfo](https://mediaarea.net/MediaInfo) to find this information*
+
+![#f5bde6](https://placehold.co/14x14/f5bde6/f5bde6.png) Audio format(s)
+:   The codec of the file's audio. *We recommend using [MediaInfo](https://mediaarea.net/MediaInfo) to find this information*
+
+    !!!info
+    If your file includes multiple audio tracks in different codecs, only list the default track's codec
+    !!!
+
+![#f0c6c6](https://placehold.co/14x14/f0c6c6/f0c6c6.png) Dual audio marker [!badge variant="warning" text="Conditional"]
+:   The marker for file using multiple audio languages/tracks
+
+![#f4dbd6](https://placehold.co/14x14/f4dbd6/f4dbd6.png) Checksum [!badge variant="success" text="Optional"]
+:   The CRC32 checksum for file verification. *We recommend using [RapidCRC](https://www.ov2.eu/programs/rapidcrc-unicode) for generating checksums*
+
++++
+
+## Folders
+
+In general, a good folder should:
+
+- List the **season number(s)** (for TV series)
+  - If you are organizing multiple seasons, these should be separated and named by season (e.g. `Season 1`, `Season 2`, ...)
```

Comment:
This isn't clear enough. Should be something as simple as "Must always mention the season number in the folder as `S0X` where X is the season number